### DISCUSSION TITLE
What does a bounce rate of "-" mean?

### DISCUSSION THREAD

**Discussion by YoranBrondsema** [plausible/analytics#2008] [Anything Else]
Some of my pages have a bounce rate of "-":

![image](https://user-images.githubusercontent.com/1374412/178410845-6c6a2b4e-d983-4f4b-8182-a2f0b1359a19.png)

What does that mean? Does it mean that there's no bounce rate for that page defined? Or the bounce rate is actually 0%? It's unclear to me.

Thank you in advance,

Yoran

**Comment by RobertJoonas** [comment]
Hi @YoranBrondsema!

Bounces for single pages are only measured for pageviews that start the session. So looking at the Top Pages report, for BR, you will see a value other than `-` only if there has been at least one entry pageview on this path. In this case, there hasn't. The 14 pageviews on this page path happened somewhere in the middle of the session. Hope this makes sense.

**Comment by YoranBrondsema** [comment]
@RobertJoonas OK I get it, thanks for the clear explanation!
